models/models_articles.js

```diff
@@ -0,0 +1,22 @@
+const db = require('../db/connection.js')
+
+function fetchArticles() {
+    return db.query("SELECT * FROM articles;")
+        .then((data) => {
+            return data.rows
+        })
+}
+
+function fetchArticle(num) {
+    return db.query(`SELECT * FROM articles
+                    WHERE article_id = ${num};`)
+        .then((data) => {
+            if (!data.rows[0]) return Promise.reject();
```

Comment:
Have a look at the notes on advanced error handling for how to handle cases where we do want an empty array back